README.md

```diff
@@ -1,14 +1,61 @@
 # Getting Started with the .NET MAUI Busy Indicator
-A quick-start project that helps you integrate a Busy Indicator control in a .NET MAUI application by adding the Syncfusion MAUI Sliders package from NuGet. This project contains simple code customizations like how to add animation, set the animation duration, adjust the header title, and change the overlay indicator background. 
-
-Refer to the following documentation about the Syncfusion .NET MAUI Busy Indicator control: 
-https://help.syncfusion.com/maui/busy-indicator/getting-started
-
-Check out this GitHub example of the Syncfusion .NET MAUI Busy Indicator control: 
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator
+This section explains the steps to configure the [.NET MAUI Busy Indicator](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Core.SfBusyIndicator.html?tabs=tabid-1) control in a real-time scenario and provides a walk-through on some of the customization features available in the control. A quick-start project that helps you integrate a Busy Indicator control in a .NET MAUI application by adding the Syncfusion MAUI Sliders package from NuGet. This project contains simple code customizations like how to add animation, set the animation duration, adjust the header title, and change the overlay indicator background. 
 
 ## Project prerequisites
-Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
+Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
 
 ## How to run this application
-To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-busy-indicator` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
\ No newline at end of file
+To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-busy-indicator` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## <a name="troubleshooting"></a>Troubleshooting ##
+### Path too long exception
+If you are facing a path too long exception when building this example project, close Visual Studio rename the repository to short, and build the project.

```

Comment:
Remove this section here.,,
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-busy-indicator/assets/95272441/41ad0af3-63c5-4883-93c1-a00a75e8e71e)

Reply:
Section removed